FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/report_builder.c

```diff
@@ -158,6 +176,24 @@ static eReportBuilderStatus prvWriteConnectionsArray( char * pcBuffer,
                                                       const Connection_t * pxConnectionsArray,
                                                       uint32_t ulConnectionsArrayLength,
                                                       uint32_t * pulOutCharsWritten );
+
+/**
+ * @brief Write task ids array to the given buffer as a JSON array.
```

Comment:
clarity _(array already implies plurality, and "ID" should be capitalized)_: 

_Write task __ID__ array to the given buffer in JSON array format._